packages/quantic/force-app/main/default/lwc/quanticResultQuickview/quanticResultQuickview.css

```diff
@@ -3,6 +3,10 @@
 .quickview__content-container {
   overflow: auto;
   border: none;
+  display: flex;
+}
+.slds-modal__content {
+  height: 90%;
```

Comment:
this was following a discussion with @erocheleau we had agreed on 90% for the modal content
otherwise it gives us something like this: 
<img width="1310" alt="image" src="https://user-images.githubusercontent.com/73316533/227341367-8237ca10-3c9e-462a-a4ed-2bbdad5f7220.png">

Reply:
It's based off the existing quickview, it takes maximum 90% of the content.